### DISCUSSION TITLE
DTM-19

### DISCUSSION THREAD

**Discussion by aubymori** [XYZenix/XYZenixThemes#21] [Suggestions & Bugs]
You should make DTM-19 based on mid-late 2019. I would love to see that.

**Comment by XYZenix** [comment]
eventually™️ (once i'm done messing with dtm-17 probably)

**Reply by TheCoreOnly1** [reply]
I think you should make the old DND and Online and Idle and Offline status from 2015-2019.
![image](https://user-images.githubusercontent.com/100382990/185210687-c76b5762-7a76-4357-a688-400c1a6ea79b.png)

**Reply by PlaayHouuse** [reply]
tbh yea

**Reply by TheCoreOnly1** [reply]
![image](https://user-images.githubusercontent.com/100382990/185221646-efe7db7a-9eb1-47c7-8bd3-7755cfb60f85.png)
this what it looks like from late 2018-2019 discord

**Reply by TheCoreOnly1** [reply]
and the people icon has been removed and replaced with discord logo.
![image](https://user-images.githubusercontent.com/100382990/185221945-0fd75b7c-741a-4bb1-a040-6855f56c3086.png)

**Reply by PlaayHouuse** [reply]
that isnt 2018 its 2019 and the home icon was replaced in mid 2018

**Reply by TheCoreOnly1** [reply]
oh, sorry if i was confused

**Reply by PlaayHouuse** [reply]
its ok

**Reply by TheCoreOnly1** [reply]
man i really wish i can go back to the good ol discord.

**Comment by PlaayHouuse** [comment]
> You should make DTM-19 based on mid-late 2019. I would love to see that.

nah early 2019 is better

**Comment by XYZenix** [comment]
dtm-19 will be based on late 2019, not early 2019. if you want early 2019, i'll do dtm-18
also, since this has gone completely off-topic, i'm locking this thread
